libsolidity/formal/SolverInterface.h

```diff
@@ -141,6 +141,7 @@ class Expression
 			{"-", 2},
 			{"*", 2},
 			{"/", 2},
+			{"mod", 2},
```

Comment:
Can these be sorted by arity? Or alphabetically? Not sure which I'd prefer ;) But maybe by arity makes more sense. This sorting should then also be applied for the operator overloadings.

Reply:
For now it's sort of sorted by theories: first Boolean operators, then arithmetic operators, then array operators.

Reply:
Ah, I see. Thanks!